### DISCUSSION TITLE
Not able to up the service.

### DISCUSSION THREAD

**Discussion by PrabhakarRTGS** [Sunbird-cQube/community#827] [General]
He Team
We have Data Center power issue in the recent period so that our cQube application down. Later we restarted Ubuntu server and its services but could not succeed. Could you plz help us to resolve the issue

**Comment by RNarmatha** [comment]
Hi Prabhakar, 

In the screenshot which you have shared looks nginx is down 
Please run the command 
**sudo docker restart nginx_app**

**Reply by PrabhakarRTGS** [reply]
![screen 3](https://github.com/Sunbird-cQube/community/assets/148041361/5b2ca161-7390-42ca-9345-02466f2b9a9e)
cannot restart container nginx_app: driver failed

**Reply by RNarmatha** [reply]
@bommalla Could you please look into this

**Reply by bommalla** [reply]
@PrabhakarRTGS Please run the below commands once.

sudo docker stop nginx_app
sudo docker start nginx_app
sudo docker restart nginx_app

**Reply by PrabhakarRTGS** [reply]
ok

**Reply by PrabhakarRTGS** [reply]
After give stop command it returns 'nginx_app' 
Then I give restart command it shows 'Error: failed to start containers: nginx_app'

**Reply by PrabhakarRTGS** [reply]
![Uploading screen 4.jpg…]()

**Comment by bommalla** [comment]
@PrabhakarRTGS Please need to check the nginx template in below mentioned path.

cqube-devops/opt/cqube/nginx
sudo vim default.conf

**Comment by bommalla** [comment]
@PrabhakarRTGS Please check the port number 80 and 443,If any other services are running in the same port numbers please kill the pid's. Please use the below command to kill the pid's

sudo netstat -ntlp
sudo kill -15 <pid>

**Comment by PrabhakarRTGS** [comment]
Hi Rajesh
Plz find the screen shot that there is no 80 & 443 port service.
________________________________

**Comment by bommalla** [comment]
Pleas use below commands to allow the port number 80,443
sudo ufw status
sudo ufw allow 80
sudo ufw allow 443

**Reply by PrabhakarRTGS** [reply]
Hi Rajesh
I tried to add rule 80 & 443 by sudo ufw allow. It is showing 'skipping adding the rule'. When I try to 'sudo ufw status' it shows allow. Screen shots attached for your consideration. Plz help us to resolve
![netstart](https://github.com/Sunbird-cQube/community/assets/148041361/f9104f84-c535-4615-ae96-684d80ab736f)
![sudo ufw status](https://github.com/Sunbird-cQube/community/assets/148041361/ff2744f1-e340-4322-877b-20771003c6b3)
![ufw allow 80](https://github.com/Sunbird-cQube/community/assets/148041361/1a3121bb-decb-4ad4-a588-8027091817f3)

**Comment by PrabhakarRTGS** [comment]
Hi Rajesh
When I try to add rule by giving command 'sudo ufw allow 80 & 443' it is showing 'skipping adding rule', but when I check status it shows allow. Kindly help us to resolve the issue. Screen shots attached for your reference.


________________________________

**Comment by PrabhakarRTGS** [comment]
Hi
Plz find the another screen shot of docker status ngins_app.

________________________________

**Comment by bommalla** [comment]
@PrabhakarRTGS Can you please tell me your availability then I can schedule call accordingly.

**Comment by PrabhakarRTGS** [comment]
Plz schedule by 10:45 today.
________________________________

**Comment by bommalla** [comment]
@PrabhakarRTGS Issue is successfully resolved.

**Comment by RNarmatha** [comment]
Issues is resolved
